examples/storage/cassandra/README.md

```diff
@@ -367,7 +367,7 @@ metadata:
   name: cassandra
   namespace: default
   resourceVersion: "323"
-  selfLink: /apis/apps/v1alpha1/namespaces/default/petsets/cassandra
+  selfLink: /apis/apps/v1alpha1/namespaces/default/statefulsets/cassandra
```

Comment:
The `selfLink` will look like this in 1.5, except that version should be `v1beta1`